22d21
<         memory > xtest_break_outside_loop
88c87
<                 xtest_break_outside_loop.write
---
>                 xtest_continue_outside_loop.write
91c90
<                       bogusForceDataize.write ((((xself).x_check_error)).apply ("break") ("outside loop"))
---
>                       bogusForceDataize.write ((((xself).x_check_error)).apply ("continue") ("outside loop"))
118c117
<                 xtest_break_outside_loop.write
---
>                 xtest_continue_outside_loop.write
122,126c121,125
<                         bogusForceDataize.write ((((xself).x_check_error)).apply ("if 0: break") ("outside loop"))
<                         bogusForceDataize.write ((((xself).x_check_error)).apply ("if 0: breaknelse:  x=1") ("outside loop"))
<                         bogusForceDataize.write ((((xself).x_check_error)).apply ("if 1: passnelse: break") ("outside loop"))
<                         bogusForceDataize.write ((((xself).x_check_error)).apply ("class C:n  if 0: break") ("outside loop"))
<                         bogusForceDataize.write ((((xself).x_check_error)).apply ("class C:n  if 1: passn  else: break") ("outside loop"))
---
>                         bogusForceDataize.write ((((xself).x_check_error)).apply ("if 0: continue") ("outside loop"))
>                         bogusForceDataize.write ((((xself).x_check_error)).apply ("if 0: continuenelse:  x=1") ("outside loop"))
>                         bogusForceDataize.write ((((xself).x_check_error)).apply ("if 1: passnelse: continue") ("outside loop"))
>                         bogusForceDataize.write ((((xself).x_check_error)).apply ("class C:n  if 0: continue") ("outside loop"))
>                         bogusForceDataize.write ((((xself).x_check_error)).apply ("class C:n  if 1: passn  else: continue") ("outside loop"))